test/e2e/node/kubelet.go

```diff
@@ -385,6 +386,102 @@ var _ = SIGDescribe("kubelet", func() {
 		}
 	})
 
+	ginkgo.Describe("Delete termination message log files when container is removed", func() {
```

Comment:
is termination log reading logic covered well by e2e tests? To ensure we are not over-removing termination logs